loki-config.yaml

```
@@ -0,0 +1,37 @@
+# Loki Configuration for Local Development
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)